packages/payload/src/fields/baseFields/baseAllLocaleStatusField.ts

```diff
@@ -0,0 +1,42 @@
+import type { Field } from '../config/types.js'
+
+export const baseAllLocaleStatusField: Field[] = [
+  {
+    name: '_localeStatus',
+    type: 'json',
+    admin: {
+      components: {
+        Cell: '@payloadcms/ui/rsc#AllLocaleStatusCell',
+      },
+      hidden: true,
+    },
+    hooks: {
+      afterRead: [
+        async (args) => {
+          const { collection, data, req } = args
+          const id = data?.id
+          if (id && collection && collection.versions) {
+            const version = await req.payload.findVersions({
```

Comment:
Can we go straight to the DB for this instead of use the local API? That will be much more performant if we don't need access control/hooks here.

Reply:
Good call - updated